src/Datadog.Trace.ClrProfiler.Native/dd_profiler_constants.h

```diff
@@ -34,7 +34,6 @@ const WSTRING env_vars_to_display[]{environment::tracing_enabled,
                                     environment::azure_app_services_cli_telemetry_profile_value};
 
 const WSTRING skip_assembly_prefixes[]{
-    WStr("Datadog.Trace"),
     WStr("MessagePack"),
```

Comment:
While you're in here, if you want, you can also remove:
- MessagePack
- Sigil
- Newtonsoft

Reply:
done in 8009608

Reply:
Thanks!